ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/synccommittee/SyncCommitteeSignatureValidator.java

```diff
@@ -100,13 +100,6 @@ public SyncCommitteeSignatureValidator(
       uniquenessKey = Optional.empty();
     }
 
-    // [IGNORE] The block being signed over (sync_committee_signature.beacon_block_root) has been
-    // seen (via both gossip and non-gossip sources).
-    if (!recentChainData.containsBlock(signature.getBeaconBlockRoot())) {
-      LOG.trace("Ignoring sync committee signature because beacon block is not known");
-      return SafeFuture.completedFuture(IGNORE);
-    }
-
```

Comment:
@ajsutton this is the bit you're talking about removing? or is there another too?

Reply:
probably just hidden, i've only made one commit. will be clearer once i can merge out all of the changes from migrate-database

Reply:
It's the equivalent code in `SignedContributionAndProofValidator` that needs to be removed.